lib/src/main/kotlin/com/swmansion/starknet/extensions/BigInteger.kt

```diff
@@ -0,0 +1,8 @@
+package com.swmansion.starknet.extensions
+
+import com.swmansion.starknet.data.types.Felt
+import java.math.BigInteger
+
+@get:JvmSynthetic
+val BigInteger.toFelt: Felt
+    get() = Felt(this)
```

Comment:
This being both public and jvmsynthetic will be a bit confusing for java users since they'll see this method in docs but will be unable to use it.

Reply:
What do you suggest? For java users this extension doesn't have a nice API.

Reply:
I don't think there's a nice way to solve this: we either have to accept that we document and expose features that are unusable (prioritize kotlin users), or perhaps change make it a standard function instead of extension (prioritize java users).

Reply:
Is there really no way to ignore this in documentation for java users? For kotlin users having this as an extension would be very useful.

Reply:
Removed it from javadoc in gradle